admin/audit.md

```diff
@@ -24,3 +23,33 @@ You can filter the logs displayed using the search filters available at the top:
 - **Resource Target**: The friendly name for the resource (e.g., the user with
   the email address **dev@coder.com**)
 - **User**: The user who performs the action
+
+## Actions
+
+The audit logs capture information about the following actions (those who
+[export Coder logs](../guides/admin/logging.md) will see this information under
+`message.fields.audit_log.action`):
+
+When reviewing Coder's audit logs, specifically, you will see the following
+actions included:
+
+- `auto_off`: Coder automatically turned on a workspace was at the preset time
```

Comment:
The wording here needs revision - I'm not sure what the intention was, but `turned on a worksapce was at the preset time` is a bit off